.github/ISSUE_TEMPLATE/bug_report_template.yml

```diff
@@ -25,15 +25,15 @@ body:
     id: reproduction
     attributes:
       label: Reproduction
-      description: Please provide a link to a repo or REPL that can reproduce the problem you ran into. Or provide the Python code below that can be run to reproduce the issue. 
+      description: Please provide a minimal example, with code, that can be run to reproduce the issue. Do NOT provide screenshots of code, or link to external repos or applications.
```

Comment:
We've started to get issues where people link to SDWUI or other applications, which are very much not minimal examples. Such issues should be opened in their repos.

Reply:
Maybe we could add a link to: https://stackoverflow.com/help/minimal-reproducible-example

It is a nice guide. Although at some point I feel we should write our own, shorter version and put it on the website or in the wiki.

Reply:
Good suggestion, will add a link to the SO guide for now